helm/pinot/values.yaml

```diff
@@ -164,6 +164,10 @@ controller:
 
   podAnnotations: {}
 
+  # set enabled as true, to automatically roll controller stateful set for configmap change
```

Comment:
Do we want to set it to true, or it should be false by default, and the comment just explain the behavior if we set it to true?

Reply:
The comment explains the behaviour. 
I think it should be `false` by default because setting it to `true` will result in a restart of the pod when the associated config map is changed.